### ISSUE TITLE
Missing InterWiki icons on PHP 8

### ISSUE THREAD

**Issue by dregad** [dokuwiki/dokuwiki#3394]
I get just an empty space, as shown in screenshot

![image](https://user-images.githubusercontent.com/449891/105926104-a017ff00-6041-11eb-991c-f34b4f207159.png)

Everything else being equal, icon is displayed as expected with PHP 7.4
![image](https://user-images.githubusercontent.com/449891/105926270-fd13b500-6041-11eb-8843-4551bcc6e67b.png)

**Comment by dregad** [comment]
Possibly a side effect of #3393

**Comment by dregad** [comment]
I confirmed that by applying the patch below the icons appear again, so it is indeed the same root cause as #3393. 

```diff
diff --git a/inc/config_cascade.php b/inc/config_cascade.php
index 61d099c6e..aa4f331a0 100644
--- a/inc/config_cascade.php
+++ b/inc/config_cascade.php
@@ -56,7 +56,8 @@ $config_cascade = array_merge(
             'screen'    => array(DOKU_CONF . 'userstyle.css', DOKU_CONF . 'userstyle.less'),
             'print'     => array(DOKU_CONF . 'userprint.css', DOKU_CONF . 'userprint.less'),
             'feed'      => array(DOKU_CONF . 'userfeed.css', DOKU_CONF . 'userfeed.less'),
-            'all'       => array(DOKU_CONF . 'userall.css', DOKU_CONF . 'userall.less')
+            'all'       => array(DOKU_CONF . 'userall.css', DOKU_CONF . 'userall.less'),
+            'speech'    => array(DOKU_CONF . 'userspeech.css', DOKU_CONF . 'userspeech.less')
         ),
         'userscript' => array(
             'default'   => array(DOKU_CONF . 'userscript.js')
```


Please let me know if this is an appropriate fix, or if I should remove the warning within css.php.

**Comment by dregad** [comment]
No longer reproducible today (tested with Jack Jackrum). Likely fixed in Igor release by #3472
